src/DetailsView/reports/components/report-sections/full-rule-detail.tsx

```diff
@@ -13,17 +13,17 @@ import { InstanceOutcomeType } from '../instance-outcome-type';
 import { OutcomeChip } from '../outcome-chip';
 import { outcomeTypeSemantics } from '../outcome-type';
 
-export type RuleDetailDeps = {
+export type FullRuleDetailDeps = {
     getGuidanceTagsFromGuidanceLinks: GetGuidanceTagsFromGuidanceLinks;
 };
 
-export type RuleDetailProps = {
-    deps: RuleDetailDeps;
+export type FullRuleDetailProps = {
+    deps: FullRuleDetailDeps;
     rule: RuleResult;
     outcomeType: InstanceOutcomeType;
 };
 
-export const RuleDetail = NamedSFC<RuleDetailProps>('RuleDetails', props => {
+export const FullRuleDetail = NamedSFC<FullRuleDetailProps>('RuleDetails', props => {
```

Comment:
The name of the NamedSFC is still `'RuleDetails'`

Reply:
:( I'll look at all the `NamedSFC` I updated/created on this PR to make sure I got them all correctly.

Reply:
Updated!